examples/dlt_statistics.rs

```diff
@@ -0,0 +1,37 @@
+use dlt_core::{
```

Comment:
in the example directory, let's also include a README.md that instructs how to run the examples